build/README.md

```diff
@@ -81,7 +81,7 @@ docker-machine create \
 # Set up local docker to talk to that machine
 eval $(docker-machine env ${KUBE_BUILD_VM})
 
-# Pin down the port that rsync will be exposed on on the remote machine
+# Pin down the port that rsync will be exposed on the remote machine
```

Comment:
This was actually grammatical IMO-- the first "on" is about the port, the second "on" is about the remote machine. However, it could read better:

```suggestion
# Pin down the port on which rsync will be exposed on the remote machine
```

or even better:

```suggestion
# Pin down the rsync port which will be exposed on the remote machine
```